### ISSUE TITLE
add elemental matrix into struct gas

### ISSUE THREAD

**Issue by jiweiqi** [DENG-MIT/Arrhenius.jl#5]
something like
```
E_C = [10, 0, 0, 1, 1, 3, 2, 3, 5, 6, 7, 0, 0, 0, 0];
E_H = [16, 1, 2, 3, 4, 5, 4, 6, 6, 6, 8, 0, 0, 1, 2];
E_N = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0];
E_O = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 2, 2];
```
This is especially useful for computing elemental conversation subspace.

**Comment by jiweiqi** [comment]
A solution mentioned in https://github.com/Cantera/enhancements/issues/49

`[([gas.n_atoms(sp, el) for sp in ['CH4', 'H2', 'H2O']], el)  for el in 'CHO']`

**Comment by jiweiqi** [comment]
This can be done in python and then save to the elementary matrix into the .npy file.

**Comment by jiweiqi** [comment]
implemented at 3a5c94d248c2ac8d278ab5160f6236ea7a1d6410

E.g.,
```
julia> gas.ele_matrix
6×40 Array{Float64,2}:
  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  2.0  …  0.0  1.0  0.0  0.0  1.0  1.0  0.0  0.0  1.0  0.0
 16.0  2.0  3.0  4.0  4.0  2.0  6.0  6.0  8.0  6.0  0.0     5.0  4.0  5.0  8.0  4.0  5.0  5.0  5.0  5.0  0.0
 10.0  0.0  1.0  1.0  2.0  2.0  3.0  6.0  7.0  5.0  0.0     3.0  3.0  4.0  4.0  5.0  5.0  5.0  6.0  6.0  0.0
  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  2.0    
  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0    
  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  …  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0    

julia> gas.elements
6-element Array{String,1}:
 "O"
 "H"
 "C"
 "N"
 "Ar"
 "He"
```
